### PR TITLE
Issue #174: Macro fixes

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

**Comment by simar7** [comment]
Still WIP, don't merge just yet.

**Comment by simar7** [comment]
@bnoordhuis, @saghul: Also can I be assigned on Issue https://github.com/libuv/libuv/issues/174 and this Pull Request?

**Comment by bnoordhuis** [comment]
> Also can I be assigned on Issue #174 and this Pull Request?

@simar7 GH doesn't let you assign to non-committers but if you want, you can claim ownership of #174 by posting a comment there.

File: test/benchmark-getaddrinfo.c [hunk 1]
```diff
@@ -83,8 +83,9 @@ BENCHMARK_IMPL(getaddrinfo) {
   ASSERT(calls_initiated == TOTAL_CALLS);
   ASSERT(calls_completed == TOTAL_CALLS);
 
-  LOGF("getaddrinfo: %.0f req/s\n",
+  fprintf(stderr, "getaddrinfo: %.0f req/s\n",
        (double) calls_completed / (double) (end_time - start_time) * 1000.0);
```
Comment (bnoordhuis) [inline test/benchmark-getaddrinfo.c]: @simar7 Style note: arguments should line up and lines should fit in 80 columns.

**Comment by txdv** [comment]
O god, thanks for removing those annoying warnings.

File: test/test-loop-handles.c [hunk 2]
```diff
@@ -235,8 +242,255 @@ static void check_cb(uv_check_t* handle) {
 static void prepare_2_cb(uv_prepare_t* handle) {
   int r;
 
-  LOG("PREPARE_2_CB\n");
+  fprintf(stderr, "%s", "PREPARE_2_CB\n");
+  fflush(stderr);
+
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```
Comment (saghul) [inline test/test-loop-handles.c]: Copy & paste error here?

**Comment by saghul** [comment]
Ping

**Comment by notorand-it** [comment]
Still here as of 1.4.2.

**Comment by Pyrohh** [comment]
> Still here as of 1.4.2.

I don't really know what you're expecting, I don't think @simar7 would mind if someone picked this up.

**Comment by simar7** [comment]
Sorry folks I've been busy for the past few weeks. I don't mind if someone picks up before I get the time to jump on this.

**Comment by saghul** [comment]
@simar7 thanks for the work! I fixed all the issues pointed out during review and added some extra fflush calls on https://github.com/libuv/libuv/pull/313.

**Comment by notorand-it** [comment]
NOT FIXED YET
Test sources have been forgotten in v1.5.0:

In file included from test/test-fs.c:23:0:
test/task.h:80:13: warning: anonymous variadic macros were introduced in C99 [-Wvariadic-macros]
 #define LOG(...)                        \
             ^
test/task.h:86:14: warning: anonymous variadic macros were introduced in C99 [-Wvariadic-macros]
 #define LOGF(...)                       \
              ^

**Comment by saghul** [comment]
@notorand-it You are mistaken. You must be using an old checkout, test/task.h does not contain that anymore: https://github.com/libuv/libuv/blob/v1.x/test/task.h#L80

**Comment by notorand-it** [comment]
Just downloaded v1.5.0, no checkout.

**Comment by saghul** [comment]
What did you download? This file: http://dist.libuv.org/dist/v1.5.0/libuv-v1.5.0.tar.gz is also correct.

**Comment by notorand-it** [comment]
@saghul
My bad, I compiled wrong version. Sorry.

**Pull request closed without merging**